src/odemis/dataio/hdf5.py

```diff
@@ -497,17 +477,35 @@ def _read_image_info(group):
                         if dataset.shape[i] == 1:
                             # To support some files saved with Odemis 2.2
                             # Now MD_OUT_WL is only mapped to EmissionWavelength
-                            logging.info("Updating MD_OUT_WL from DimensionScaleC, only supported in backward compatible mode")
+                            logging.info("Updating MD_OUT_WL from DimensionScaleC, "
+                                         "only supported in backward compatible mode")
                             md[model.MD_OUT_WL] = (pn[0], pn[0] + pn[1])
                         else:
                             md[model.MD_WL_POLYNOMIAL] = pn
 
             elif dim.label == "T" and dim:
+                # add time list to metadata, containing info on time stamp per px if available
                 if dim[0].shape == (dataset.shape[i],):
                     md[model.MD_TIME_LIST] = map(float, dim[0][...].tolist())
```

Comment:
I did only copy and paste ;) 
Is dim[0][...] special to access a h5 dataset  or is that a general method in python and if yes what is it called so I can search for it.

Reply:
A little of both. `...` is a special Python construct call `Ellipsis`. In numpy, it means something like "insert as many full slices (:) to extend the multi-dimensional slice to all dimensions". I think here it forces to always get an array (as otherwise it might return just a scalar, if there is no dimension).  With h5py, I don't recall why we need the `[0]`, but it's probably still needed.

Reply:
BTW, I know you only copy-pasted... and that code is probably from me ;-)